Assets/VRTK/Source/Scripts/Pointers/PointerRenderers/VRTK_BasePointerRenderer.cs

```diff
@@ -154,6 +154,7 @@ public virtual void InitalizePointer(VRTK_Pointer givenPointer, VRTK_PolicyList
         public virtual void ResetPointerObjects()
         {
             DestroyPointerObjects();
+            CreatePointerOriginTransformFollow();
```

Comment:
Thinking about this more, I should have a `DestroyPointerRendererObjects` here too, to prevent multiple from being created